tests/installation/npmTest.ts

```diff
@@ -60,6 +60,37 @@ const expect = _expect.extend({
         `   actual: ${[...downloaded].sort().join(', ')}`,
       ].join('\n'),
     };
+  },
+  toHaveListedBrowsers(received: string, browsers: ('chromium' | 'chromium-headless-shell' | 'firefox' | 'webkit' | 'winldd' |'ffmpeg')[]) {
+    if (typeof received !== 'string')
+      throw new Error(`Expected argument to be a string.`);
+
+    // Extract browser names from the received string
+    const listed = new Set();
+    let index = 0;
+    while (true) {
+      const match = received.substring(index).match(/Browser: (chromium|chromium-headless-shell|firefox|webkit|winldd|ffmpeg)/im);
+      if (!match)
+        break;
+      listed.add(match[1].replace(/\s/g, '-').toLowerCase());
+      index += match.index + 1;
+    }
+
+    const expected = new Set(browsers);
```

Comment:
Just leverage toEqual? Some modification of `expect([...listed].sort()).toEqual(browsers.sort());`